backend/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+bitcoin = { version = "0.32", features = ["serde"] }
```

Comment:
I'll have a look if re-exporting bitcoin with serde through rawtx is possible.

To ensure the versions of `bitcoin` and `rawtx-rs::bitcoin` are compatible, I've been using the rawtx-rs rexport of `bitcoin` (i.e. `rawtx-rs::bitcoin`). If possible, I'd like to keep it that way.

Reply:
Sure, we can do that but I would have to update rawtx's bitcoin dep to use the serde feature. I didn't want to update any packages yet for this proof of concept. I think we can actually get rid of rawtx after this PR, since we have all the info from the JSON response so we don't need to serialize the hex.

Reply:
>  I think we can actually get rid of rawtx after this PR, since we have all the info from the JSON response so we don't need to serialize the hex.

Hm I would avoid getting rid of rawtx-rs anytime soon. Next to input and output type detection, rawtx-rs can:
- count sigops
- detect multisig
- extract information about pubkeys and signatures
- detect op_return flavors
- detect inscriptions
- ...

I wouldn't want to re-implement that here